### ISSUE TITLE
FoW issue/bug exposing from Token

### ISSUE THREAD

**Issue by thecko** [RPTools/maptool#781]
Hello,

I have a problem with Maptool FoW exposing. Not sure if it's a bug or something I'm missing. This behaviour happened in at least 3 computers (all Windows based), 2 are W10, 1 is W8, so I'm assuming it's not something related to the equipment.

Basically I create a map, import an image as background, add a "hero" token as PC and then I create the CBV by creating a giant blue box and then "carve" the zones (see image).

Then, if I activate the FoW, select the token and then I try to Expose visible area (either right click on the Token or Control + I with it selected) it does nothing.

Strangely, this does not happen always. If in the same campaign I create a new map and repeat the process, some times it works.

I'm trying to figure out if it's a bug or something related to options.

We've been in Discord trying to reproduce it and it was not possible. But I've seen that copying or exporting and importing the map file does not solve it.

You can find a zip file with the campaign here:
https://drive.google.com/open?id=1JaKq1ITJ7PRM6V1m4rqGkkADkueaLNyb

It contains the image I use as background, the campaign file and the bugged map exported.

Many thanks

**Comment by Phergus** [comment]
The issue is that your map has a Distance per Cell setting of 50000.  So with the default light sources like a torch (20') you were only exposing 0.02 pixels.  From the map name I'm guessing Spain?  If so you are probably being hit by the bug reported in #507.

Map Menu -> Edit Map and clear the Distance per Cell entry.  Then just put in 5 with no decimal point and click on OK.  That will fix the map.   Then go into Edit -> Preferences then Interactions -> Map and check the New Map Distance per Cell setting.  Probably need to change that to 5 as well.

**Comment by thecko** [comment]
I repost here what we managed to discover on Discord.

I never changed that value. After some tests, we saw that after changing the value back to a value of 50, accept and going back to map > Edit map now it shows 500. Repeating again does not increase again the value but I'm pretty sure there's a way to reproduce it again and that because it went to 50000.

Later on, after another comment and test, it seems it happens when Spanish locale is loaded (not tested with other languages). Setting English seems to avoid the error.
